app/controllers/ncr/work_orders_controller.rb

```diff
@@ -21,9 +21,26 @@ def edit
     end
 
     def update
+      @model_instance.assign_attributes(permitted_params)
       @model_instance.modifier = current_user
 
-      super
+      @model_changing = false
+      @model_instance.validate
+
+      if self.errors.empty?
+        if self.attribute_changes?
+          ProposalUpdateRecorder.new(@model_instance).run
```

Comment:
it's situations like these when the `UseCaseController` seems to be more of a barrier than a help...that is, unless we want to have this same behavior for `Gsa18f::Procurement` as well...(behavior being creating a comment for updates to a proposal). cc @phirefly